scripts/backend_test_shards.json

```diff
@@ -209,7 +209,8 @@
         "core.domain.app_feedback_report_constants_test",
         "scripts.linters.warranted_angular_security_bypasses_test",
         "scripts.create_expression_parser_test",
-        "scripts.run_frontend_tests_test"
+        "scripts.run_frontend_tests_test",
+        "scripts.contributor_dashboard_debug_test"
```

Comment:
Shard 3 is taking >30 mins to complete, so can you add this test to a shard that's running fast to better balance the load? This requirement is documented in more detail here: https://github.com/oppia/oppia/wiki/Backend-tests#adding-new-tests-to-shards